Source/Common/IGListBatchUpdateData.mm

```diff
@@ -47,7 +47,7 @@ + (void)cleanIndexPathsWithMap:(const std::unordered_map<NSUInteger, IGListMoveI
                     indexPaths:(NSMutableSet<NSIndexPath *> *)indexPaths
                        deletes:(NSMutableIndexSet *)deletes
                        inserts:(NSMutableIndexSet *)inserts {
-    for (NSIndexPath *path in [indexPaths copy]) {
```

Comment:
I made a similar change to another indexSet method and undo it, missed this one.